CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Features
 - Add `color` prop to `Text` component @Bugaa92 ([#597](https://github.com/stardust-ui/react/pull/597))
 - Add `color` prop to `Header` and `HeaderDescription` components @Bugaa92 ([#628](https://github.com/stardust-ui/react/pull/628))
+- [Teams Theme] Add and export new compose icons @joheredi ([#639](https://github.com/stardust-ui/react/pull/639))
```

Comment:
lets stick to the naming convention that is used for the rest of CHANGELOG and make it

> Export compose icons by Teams theme @joheredi